examples/queries.md

```
@@ -108,6 +108,32 @@ args: []
+    ).
+    KeysetAfter(120, "2025-01-01 00:00:00").
+    Limit(20)
+
+sql, args := page.Build()
+```
+
+**Saída gerada**
+```
+SELECT id, created_at FROM posts WHERE ((id > ? OR (id = ? AND created_at < ?))) ORDER BY id ASC, created_at DESC LIMIT 20
+args: [120 120 "2025-01-01 00:00:00"]
+```
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)